lib/security_blog.js

```diff
@@ -331,7 +331,12 @@ export default class SecurityBlog {
         affectedVersions.add(affectedVersion);
       }
     }
-    return Array.from(affectedVersions).join(', ');
+    const parseToNumber = str => str.match(/[\d.]+/g)[0];
```

Comment:
This doesn't actual parse to a number? (i.e. this is parsing to a string comprising of digits.)

Reply:
Yeah, I forgot the `+`.